demos/react-jsnation/tests/app.spec.tsx

```diff
@@ -0,0 +1,21 @@
+// @vitest-environment jsdom
+import { cleanup, render, screen, within } from "@testing-library/react";
+import { afterEach, describe, expect, test } from "vitest";
+import { App } from "@starbeam-demos/jsnation";
+
+describe("react-jsnation", () => {
```

Comment:
These smoke tests are intentionally very basic - Just checking some of the text rendered in the app to ensure the app is loading properly.